test/references/DataOneV1 Ecology.txt

```diff
@@ -10,9 +10,9 @@ register(DataDep(
 	Domesticated species are impacted in unintended ways during domestication and breeding. Changes in the nature and intensity of selection impart genetic drift, reduce diversity, and increase the frequency of deleterious alleles. Such outcomes constrain our ability to expand the cultivation of crops into environments that differ from those under which domestication occurred. We address this need in chickpea, an important pulse legume, by harnessing the diversity of wild crop relatives. We document an extreme domestication-related genetic bottleneck and decipher the genetic history of wild populations. We provide evidence of ancestral adaptations for seed coat color crypsis, estimate the impact of environment on genetic structure and trait values, and demonstrate variation between wild and cultivated accessions for agronomic properties. A resource of genotyped, association mapping progeny functionally links the wild and cultivated gene pools and is an essential resource chickpea for improvement, while our methods inform collection of other wild crop progenitor species.
 	
 	Please cite this paper:
-	von Wettberg, Eric J. B. Chang, Peter L Başdemir, Fatma Carrasquila-Garcia, Noelia Korbu, Lijalem Moenga, Susan M. Bedada, Gashaw Greenlon, Alex Moriuchi, Ken S. Suryawanshi, Vasantika Cordeiro, Matilde A Noujdina, Nina V. Dinegde, Kassaye Negash Abbas Shah Sani, Syed Gul Getahun, Tsegaye Vance, Lisa Bergmann, Emily Lindsay, Donna Mamo, Bullo Erena Warschefsky, Emily J. Dacosta-Calheiros, Emmanuel Marques, Edward Yilmaz, Mustafa Abdullah Cakmak, Ahmet Murat Rose, Janna Migneault, Andrew Krieg, Christopher P. Saylak, Sevgi Temel, Hamdi Friesen, Maren L. Silor, Eleanor Akhmetov, Zhaslan Ozelik, Huseyin Kholova, Jana Can, Canan Gaur, Pooran Yildirim, Mehmet Sharma, Hari Vadez, Vincent Tesfaye, Kassahun Woldemedhin, Asnake Fikre Tar'an, Bunyamin Aydogan, Abdulkadir Bukun, Bekir Penmetsa, R. Varma Berger, Jens Kahraman, Abdullah Nuzhdin, Sergey V. Cook, Douglas R. (2018) Data from: Ecology and genomics of an important crop wild relative as a prelude to agricultural innovation https://doi.org/10.1038/s41467-018-02867-z
+	Von Wettberg, E. J. B., Chang, P. L., Başdemir, F., Carrasquila-Garcia, N., Korbu, L. B., Moenga, S. M., … Cook, D. R. (2018). Ecology and genomics of an important crop wild relative as a prelude to agricultural innovation. Nature Communications, 9(1). doi:10.1038/s41467-018-02867-z
 	as well as this dataset:
-	https://doi.org/10.5061/dryad.74699
+	Von Wettberg, E. J. B., Chang, P. L., Başdemir, F., Carrasquila-Garcia, N., Korbu, L., Moenga, S. M., … Cook, D. R. (2018). Data from: Ecology and genomics of an important crop wild relative as a prelude to agricultural innovation [Data set]. Dryad Digital Repository. https://doi.org/10.5061/dryad.74699
```

Comment:
I tried on a VPN, and using Postman which gives me this as a result. But Travis doesn't.